zkevm-circuits/src/tx_circuit.rs

```diff
@@ -1932,4 +1932,21 @@ mod tx_circuit_tests {
         )
         .is_err(),);
     }
+
+    #[test]
+    fn tx_circuit_to_is_zero() {
+        const MAX_TXS: usize = 1;
+        const MAX_CALLDATA: usize = 32;
+
+        let chain_id: u64 = mock::MOCK_CHAIN_ID.as_u64();
+        let mut tx = mock::CORRECT_MOCK_TXS[0].clone();
+        tx.to = Some(AddrOrWallet::from(address!(
+            "0x0000000000000000000000000000000000000000"
+        )));
```

Comment:
```suggestion
        tx.to = Some(AddrOrWallet::from(Address::zero()));
```

Reply:
Not sure if `address!` macro was imported

Reply:
fixed.